src/Aspire.Hosting.Nats/Aspire.Hosting.Nats.csproj

```diff
@@ -8,6 +8,14 @@
     <PackageIconFullPath>$(SharedDir)nats-icon.png</PackageIconFullPath>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <MinCodeCoverage>7</MinCodeCoverage>
```

Comment:
This can be removed as it gets overridden below.

Reply:
Oh! Thanks for this, I totally missed it.

Reply:
FWIW, the top one is the one we want - that's the one the validation script reads. The botton one is the one redundant.
Since these aren't used by MSBuild, there's no overriding involved.